docs/user/tutorials/quickstart.md

```diff
@@ -0,0 +1,48 @@
+# Quickstart
+
+!!! note "Docker/Podman is required"
+
+    The first time the script is run, a docker container with openapi-generator-cli is downloaded. All
+    subsequent runs will re-use the container that was downloaded on the initial run.
+
+First, clone the repo into your machine. It'll be used as the workspace for the bindings generation.
+Then, choose methods A or B.
+
+In any case, the final packages are stored in the `{plugin_label}-client` directory.
+
+```bash
+git clone https://github.com/pulp/pulp-openapi-generator
+```
+
+## A) Generate from running Pulp
+
+Configure the settings for you Pulp instance using [`PULP_URL`](#pulp_url) and [`PULP_API_ROOT`](#pulp_api_root).
+The generator will fetch the openapi schema directly from it to generate the client.
+
+The command below will generate a ruby client for pulp_rpm at `./pulp_rpm-client/`.
+
+```bash
+PULP_URL="http://localhost:5001" ./generate.sh pulp_rpm ruby
```

Comment:
Generate is the thing we keep around for the old branches.
I think, we should not document it in favor of the more modern gen-client command.

Reply:
Oh, ok. Didnt know about that.

Reply:
I should have probably put more effort in warning messages there.

Also "the old/new" way is a matter of how the plugin template uses it, right? Maybe we can consolidate this process. But maybe there other places still...